### ISSUE TITLE
systemd service file broken

### ISSUE THREAD

**Issue by errhammr** [yggdrasil-network/yggdrasil-go#809]
Hello there!

I just installed yggdrasil 0.4.0 on a debian server. Running `yggdrasil -useconffile /etc/yggdrasil.conf` as root worked just fine but the systemd service just wouldn't start:
```
panic: listen tcp 0.0.0.0:443: bind: permission denied
```

I'm running systemd version `241-7~deb10u7` on `Debian GNU/Linux 10 (buster)` with kernel `Linux 4.19.0-16-amd64 #1 SMP Debian 4.19.181-1 (2021-03-19)`.

After looking around on the internet I came across a [similar problem in a different software](https://serverfault.com/a/899964) and that gave me a clue how to fix it. In `/etc/systemd/system/yggdrasil.service` (which is [this file in this repo](https://github.com/yggdrasil-network/yggdrasil-go/blob/master/contrib/systemd/yggdrasil.service)) I had to append `CAP_NET_BIND_SERVICE` to the line starting with `CapabilityBoundingSet`.

Line before editing:
```
CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_RAW
```

Line after editing:
```
CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_RAW CAP_NET_BIND_SERVICE
```

That fixed the issue for me and the systemd service was able to start.

I don't know if that is the proper way of fixing this issue or if something different would be a more elegant approach. So I won't be filing a PR for this. I just wanted to let you know about the problem and how it can be fixed.

**Comment by zander** [comment]
The default port is > 1024, which does not require the permission and it would not be a good idea to require the permission for the majority that does not need it. On that ground its not "broken".

What might be nice is to add a comment indicating a solution for those that want it, and definitely beneficial would be to have a note in the installation documentation.

**Comment by cofob** [comment]
It's ugly, and may not be safe, but it works.

`/etc/systemd/system/yggdrasil.service`
```
[Unit]
Description=yggdrasil
Wants=network.target
After=network.target

[Service]
User=root
Group=root
ExecStart=/bin/bash /root/start_ygg
Restart=always
TimeoutStopSec=5

[Install]
WantedBy=multi-user.target
```

`/root/start_ygg`
```
#!/bin/bash
yggdrasil -useconffile /etc/yggdrasil.conf
```

**Comment by neilalexander** [comment]
Should be fixed by #813.
